hackathon_site/hardware/api_filters.py

```diff
@@ -24,6 +25,13 @@ class IncidentFilter(filters.FilterSet):
     hardware_id = filters.NumberFilter(field_name="order_item__hardware__id")
     team_id = filters.NumberFilter(field_name="order_item__order__team__id")
 
+class OrderItemFilter(filters.FilterSet):
+    queryset = OrderItem
+    serializer_class = OrderItemListSerializer
+
+    hardware_id = filters.NumberFilter(field_name="order__id")
```

Comment:
change this variable name to order_id and we would want to be able to put multiple order ids so IntegerCSVFilter might be a be a better option, look at OrderFilter for example.